.github/workflows/test.yaml

```
@@ -34,3 +34,17 @@ jobs:
         run: yarn test
         env:
           CI: true
+
+    name: inclusive-naming-check
```

I think you want to define it as separate job? Alongside the `build` above?

```suggestion
  inclusive-naming-check
```